src/main/resources/datainfo.properties

```diff
@@ -285,4 +285,7 @@ keycloak.credentials.secret=
 auth.realm: master
 auth.client-id: admin-cli
 auth.client-secret:
-auth.base-url: https://auth.openclinica-dev.io/auth
\ No newline at end of file
+auth.base-url: https://auth.openclinica-dev.io/auth
+
+# Discrepancy Note
+dicrepancyNote.DisplayId.GenerationLength=9
```

Comment:
rename this to "discrepancyNote.displayId.length" to be somewhat consistent with other property names